README.md

```diff
@@ -1,6 +1,209 @@
+    ·
+    <a href="https://github.com/rasmusmarak/TOSS/issues">Request Feature</a>
+  </p>
+</p>
+
+
+
+### Built With
+
+This project is based on:
+
+- [polyhedral-gravity-model](https://github.com/esa/polyhedral-gravity-model) a fast, parallelised version of the polyhedral gravity model implemented in C++ along with a python interface.
+- [DESolver](https://github.com/Microno95/desolver) a python library fro solving Initial Value Problems using numerical integrators. 
+- [PyGMo](https://esa.github.io/pygmo2/index.html) a scientific python library for massively parallel optimization managed by the Advanced Concepts Team at the European Space Agency
+
+For more details than provided by TOSS on these libraries, please refer to their docs.
+
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+This is a brief guide how to set up TOSS.
+
+### Installation
+
+### conda
+The recommended way to install TOSS is via [conda](https://docs.conda.io/en/latest/) / [mamba](https://github.com/conda-forge/miniforge#mambaforge) using
+
+```
+conda install toss -c conda-forge
+
+```
+
+### Building from source
+
+Alternatively, to use the latest code from this repository make sure you have all the requirements installed and then clone the [GitHub](https://github.com/rasmusmarak/TOSS.git) repository as follows ([Git](https://git-scm.com/) required):.
+
+```
+git clone https://github.com/rasmusmarak/TOSS
+```
+
+To install TOSS you can use [conda](https://docs.conda.io/en/latest/) as follows:
+
+```
+cd TOSS
+conda env create -f environment.yml
+```
+
+This will create a new conda environment called `toss` and install the required software packages.
+To activate the new environment, you can use:
+
+```
+conda activate toss
+```
+
+
+### Test
+
+After cloning the repository, developers can check the functionality of TOSS by running the following command in the `TOSS/tests` directory:
+
+```sh
+pytest
+```
+
+<!-- USAGE EXAMPLES -->
+
+## Usage
+
+### Config
+
+TOSS uses a central config file which is passed through the entire program. The default config parameters can be seen [here](https://github.com/rasmusmarak/TOSS/blob/2501351468eb362fa9ec8217b2fc426db37fca9a/toss/resources/default_cfg.toml). 
+
+**[Coming Soon]** Practical usage of the config files will soon be added here through an explanatory notebook.
+
+### Use Case 1: Trajectory Optimization of a Swarm Orbiting 67P/Churyumov-Gerasimenko Maximising Gravitational Signal
+In this case, the aim is to compute a set of trajectories corresponding to a spacecraft swarm of n spacecraft, each with m impulsive maneuvers. The simulation is done within the context of a prospective mission around the comet 67P/Churyumov-Gerasimenko and the scientific target of maximising the measured gravitational signal. For details on mission parameters, optimization structure and fitness functions, see the corresponding paper on [ArXiv](https://doi.org/10.48550/arXiv.2306.01602).
```

Comment:
I would suggest to use this https://citation-file-format.github.io/ , would also mention the paper higher up somewhere

Reply:
Added the cff-file in PR #54. Maybe we could merge that branch into this one for simplicity, and then I'll change the referencing in this PR directly before merging to main?

Reply:
We can but it seems you branched out of main in #54 , thus it would be easier to merge them individually I think